tests/console/console.go

```diff
@@ -47,16 +47,16 @@ var (
 	shellFail    = regexp.MustCompile(RetValue("[1-9].*"))
 )
 
-// VmiConsoleExpectBatch runs the batch from `expected` connecting to the `vmi` console and
+// ExpectBatch runs the batch from `expected` connecting to the `vmi` console and
 // wait `timeout` for the batch to return.
 // NOTE: there is a safer version that validates sended commands `CheckForTextExpecter` refer to it about limitations.
-func VmiConsoleExpectBatch(vmi *v1.VirtualMachineInstance, expected []expect.Batcher, timeout time.Duration) error {
+func ExpectBatch(vmi *v1.VirtualMachineInstance, expected []expect.Batcher, timeout time.Duration) error {
```

Comment:
`CheckForTextExpecter` has the same functionality as `ExpectBatch` besides the send validation, but the name of the methods doesn't reflect.
Also the new name is confusing, since we already have `ExpectBatchWithValidatedSend`. And now we have `ExpectBatch` I would think the only difference between the methods is the second one is doing validation on the send.

My suggestion -
VmiConsoleExpectBatch -> ExpectBatchOnVmi
CheckForTextExpecter => ExpectBatchOnVmiWithValidatedSend

Reply:
I think that adding the `OnVmi` in the function names is redundant. We are under the `console` which is only relevant for VMI/s and in general, expecters also run only on VMI/s.

So how about CheckForTextExpecter => ExpectBatchWithValidatedSend ?

Reply:
> I think that adding the `OnVmi` in the function names is redundant. We are under the `console` which is only relevant for VMI/s and in general, expecters also run only on VMI/s.
> 
> So how about CheckForTextExpecter => ExpectBatchWithValidatedSend ?

what about something simple
`SafeExpect`? all the other words are redundant imo

Reply:
I tried not to diverge too much from the original, dropping just the vmi and console. I am fine with this shorter one as well.

Reply:
I took the `Safe` and the `Batch`, seems like a good compromise.